styles.css

```
@@ -0,0 +1,76 @@
+* {
+  Box-sizing: border-box; }
```

Keep your CSS lowercase please 🙏🏻

```suggestion
  box-sizing: border-box; }
```